packages/ember-glimmer/tests/integration/custom-component-manager-test.js

```diff
@@ -0,0 +1,252 @@
+import { moduleFor, RenderingTest } from '../utils/test-case';
+import { strip } from '../utils/abstract-test-case';
+import { Component } from '../utils/helpers';
+
+import { Object as EmberObject } from 'ember-runtime';
+import { set, setProperties, computed } from 'ember-metal';
+import { GLIMMER_CUSTOM_COMPONENT_MANAGER } from 'ember/features';
+import { componentManager, CustomComponentManager } from 'ember-glimmer';
+import { getChildViews } from 'ember-views';
+
+const MANAGER_ID = 'test-custom';
+
+const CustomComponent = componentManager(EmberObject.extend(), MANAGER_ID);
+
+if (GLIMMER_CUSTOM_COMPONENT_MANAGER) {
+  moduleFor('Components test: curly components with custom manager', class extends RenderingTest {
+    /*
+     * Helper to register a custom component manager. Provides a basic, default
+     * implementation of the custom component manager API, but can be overridden
+     * by passing custom hooks.
+     */
+    registerCustomComponentManager(overrides = {}) {
+      let options = Object.assign({
```

Comment:
Needs to be updated to use assign from ember-utils (Object.assign isn’t available on all tested platforms).